### ISSUE TITLE
FireTV Cube Unable to Retrieve List of Running Apps

### ISSUE THREAD

**Issue by tommyjlong** [JeffLIrion/python-androidtv#292]
FireTV Cube running Fire OS 7.2.4.2
In Home Assistant, the Source List in the Media Card only shows the current running App.
I had the same problem with a FireTV Stick (which I no longer use).

In my looking around at the code, what I think is the problem is the ADB Shell command
that FireTV uses (from constants.py): 
```
#: Get the running apps for a Fire TV device
CMD_RUNNING_APPS_FIRETV = "ps | grep u0_a"
```
When I change this to the following (which is the same as for the Android Device): 
```
#: Get the running apps for a Fire TV device
CMD_RUNNING_APPS_FIRETV = "ps -A | grep u0_a"
```
The running Apps list is actually retrieved!!!

**Comment by SamB-GB** [comment]
I had the same issue and the only way I could getting the Firestick 4K Max to work was to re-add it and choose the device type as Android rather than FireTV, not really sure what the full consequences of that are but that does fix the issue with the sources.

![Snag_694da4](https://user-images.githubusercontent.com/15860857/213590871-cc3ec528-3aba-48f8-8322-6ec300e46cea.png)
![Snag_683a8e](https://user-images.githubusercontent.com/15860857/213590872-5666dcf7-18cd-41e8-b64f-317335530a65.png)

**Comment by tommyjlong** [comment]
Yeah I was just hoping for a one line change to `CMD_RUNNING_APPS_FIRETV = "ps -A | grep u0_a"`
